ihatemoney/templates/list_bills.html

```diff
@@ -117,6 +117,7 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
         <table id="bill_table" class="col table table-striped table-hover table-responsive-sm">
             <thead>
                 <tr><th>{{ _("When?") }}
+                </th><th>{{ _("Bill Type") }} 
```

Comment:
Rather than adding a column, what about adding an info next to the name, or just use the "for what?" field?

Reply:
I actually just removed the bill type column now. The fact that it's a settlement is indeed clear from the "for what" field. Now it looks like this:
![image](https://github.com/spiral-project/ihatemoney/assets/21120212/a6efa4d3-0734-473a-94c7-767c6bc90502)